src/engine/ruleset/decoders/fortinet/fortimail.yml

```diff
@@ -0,0 +1,205 @@
+---
+name: decoder/fortinet-fortimail/0
+
+metadata:
+  module: fortinet
+  title: Decoder for Fortinet fortimail
+  description: Decoder for Fortinet fortimail
+  version: [6.0.x, 6.2.x]
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/09
+  references:
+    - https://docs.fortinet.com
+
+sources:
+  - decoder/queue-localfile/0
```

Comment:
Can't Fortinet logs come through Syslog?

Reply:
in this case syslog does not apply, we verify it.